pathfinder/tests/test_blockchain_integration.py

```diff
@@ -76,14 +75,14 @@ def test_pfs_with_mocked_client(
 
     # check that deposits and transfers got registered
     for index, (
-        p1_index,
+        _p1_index,
         p1_deposit,
-        p1_transferred_amount,
-        p1_fee,
-        p2_index,
+        _p1_transferred_amount,
+        _p1_fee,
+        _p2_index,
         p2_deposit,
-        p2_transferred_amount,
-        p2_fee
+        _p2_transferred_amount,
+        _p2_fee,
     ) in enumerate(channel_descriptions_case_1):
         channel_identifier = channel_identifiers[index]
```

Comment:
don't you also need a comma here then?

Reply:
Why? this is just a statement

Reply:
ok so no commas behind statements but behind arguments, got it